conda/recipes/custreamz/meta.yaml

```diff
@@ -32,8 +32,8 @@ requirements:
     - python
     - streamz
     - cudf {{ version }}
-    - dask>=2021.11.1,<=2022.01.0
-    - distributed>=2021.11.1,<=2022.01.0
+    - dask>=2021.11.1
+    - distributed>=2021.11.1
```

Comment:
Probably out of the scope of this PR but would we ever want cuDF nightlies to be built against Dask nightlies? Think this should be configurable by optionally adding `--channel dask/label/dev` to the conda build step

Reply:
will be opening a PR for it in https://github.com/rapidsai/gpuci-build-environment/blob/b5e7272ba167affd8bf0fd1d2248f58478725768/rapidsai/base-runtime.Dockerfile#L35-L45

Reply:
That would give us a Docker image with the Dask and cuDF nightlies coexisting right?

I'm more thinking of a scenario where a change made to dask-cudf is dependent on some upstream change in Dask that is merged in but not in a stable release - in that case, `conda install`ing dask-cudf would pull in an outdated Dask and result in some breakage (unless I'm missing something).

The change I'm thinking of would involve altering the `CONDA_BUILD_ARGS` in these commands:

https://github.com/rapidsai/cudf/blob/8d2a9cc8fafd40d8af2c5fba11d394d07b619833/ci/cpu/build.sh#L103-L113

Reply:
This commit https://github.com/rapidsai/cudf/pull/10182/commits/8b717f95c80e5a7bb6a6190c2a060fb60617bb27 along with https://github.com/rapidsai/gpuci-build-environment/pull/238 should achieve what you are saying @charlesbluca